Lib/json/tool.py

```diff
@@ -24,20 +24,30 @@ def main():
                         help='a JSON file to be validated or pretty-printed')
     parser.add_argument('outfile', nargs='?', type=argparse.FileType('w'),
                         help='write the output of infile to outfile')
+    parser.add_argument('--indent', default=4,
```

Comment:
declare the type here for automatic cast:
```python
type=int
```

Reply:
Avoiding int type is intentional, since `json.dumps` can also accept integer number of spaces, or arbitrary user-provided string. This is a feature.

Reply:
ack! I take notes